src/commands/deploy/utils.ts

```diff
@@ -71,6 +82,85 @@ export async function getOrCreateEmptyProject(
     return { projectId: project.id, projectEnvId: projectEnv.id };
 }
 
+export async function readOrAskConfig(configPath: string): Promise<YamlProjectConfiguration> {
+    const configIOController = new YamlConfigurationIOController(configPath);
+    if (!existsSync(configPath)) {
+        const name = await readOrAskProjectName();
+
+        let region = regions[0].value;
+        if (process.env["CI"] !== "true") {
```

Comment:
use `isCI()` instead
https://github.com/Genez-io/genezio/blob/dev/src/utils/process.ts#L47

```suggestion
        if (!isCI()) {
```